app/models/course/survey.rb

```diff
@@ -56,4 +57,17 @@ def include_in_consolidated_email?(event)
     email_enabled = course.email_enabled(:surveys, event)
     email_enabled.regular || email_enabled.phantom
   end
+
+  # @override ConditionalInstanceMethods#permitted_for!
+  def permitted_for!(course_user)
```

Comment:
Hmm anything required here?

Reply:
AFAIK no; this method is used by `Achievement` to mutate `course_users`, but `Survey` is similar to `Assessment` since they both don't have such an association relation and thus this method is left empty (same as the implementation in `Assessment`).